src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -2616,6 +2540,20 @@ class ConsumerGroupParameterSegment(BaseSegment):
     )
 
 
+@exasol_dialect.segment()
+class DropConsumerGroupSegment(BaseSegment):
+    """A `DROP CONSUMER GROUP` statement.
+
+    https://docs.exasol.com/sql/consumer_group.htm
+    """
+
+    type = "drop_consumer_group_statement"
```

Comment:
A lot of the other segments in the exasol dialect flag the statement type like this:
```
is_ddl = False
is_dml = False
is_dql = False
is_dcl = True
```
Should we also add it in here?

Also do you know if these get used anywhere? (haven't seen them in the other dialects)

Reply:
I implemented them as I introduced the Exasol dialect. The idea behind this was to use this information after parsing for other programs (e.g. our deployment tool which now works with `sqlfluff` under the hood). 

It could be discussed if we should remove this attributes.

A consumer group fits in none of these attributes.

Reply:
Ah nice, thanks for clarifying! tbh I don't think there's any need to remove those attributes anytime soon was just curious 😄